packages/stacked_generator/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 0.5.1
 
+- Adds `orElse` handler in the logger `realFirstLine` variable to fix [issue #455](https://github.com/FilledStacks/stacked/issues/455)
```

Comment:
Please move this to a new version in the change log and update the pubspec.yaml to match

Reply:
Updated 👍🏻